src/seamless_communication/segment/silero_vad.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# MIT_LICENSE file in the root directory of this source tree.
+
+from __future__ import annotations
+
+from argparse import Namespace
+import torch
+
+
+class SileroVADSegmenter:  # type: ignore
+    def __init__(self, args: Namespace) -> None:
```

Comment:
I wonder why you provide a Namespace instead of more traditional keyword args?
Is there any advantage in doing so?

Reply:
Just to prevent users from passing unexpected arguments, but it isn't necessary here. It would also work to use kwargs or function parameters, my intention was to maintain style consistency with similar agent classes such as SileroVADAgent